modules/tooltip/react/lib/Tooltip.tsx

```diff
@@ -1,74 +1,69 @@
 import * as React from 'react';
-import styled from '@emotion/styled';
-import {borderRadius, colors, spacing, type} from '@workday/canvas-kit-react-core';
-import {TransformOrigin, getTranslateFromOrigin} from '@workday/canvas-kit-react-common';
-import {keyframes} from '@emotion/core';
+import innerText from 'react-innertext';
 
-export interface TooltipProps {
+import {getTransformFromPlacement, Popper, Placement} from '@workday/canvas-kit-react-common';
+
+import {TooltipContainer} from './TooltipContainer';
+import {useTooltip} from './useTooltip';
+
+export interface TooltipProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'title'> {
+  /**
+   * Determines the tooltip type for accessibility.
+   * * `label`: Sets the accessible name for the wrapped element. Use for icons or if tooltip `title` prop is the same as the text content of the wrapped element. E.g. IconButtons or Ellipsis tooltips.
+   * * `describe`: Sets `aria-describedby` of the wrapped element. Use if the tooltip has additional information about the target.
+   *
+   * **Note**: Assistive technology settings may ignore "aria-describedby". Consider alternative
+   * way to inform a user of additional important information.
+   * @default 'label'
+   */
+  type?: 'label' | 'describe';
```

Comment:
Nit: required props on top

Reply:
I never really thought about it this way haha. Maybe we should discuss as a team.

I guess I look at it as "all of the things that are guaranteed to be defined come first".